### PR TITLE
WIP: Render task details in DiscussView

### PR DESCRIPTION
**Pull request by elimisteve**
@MartyTheeMartian @Moe-Shoman Should work except for the definition of the custom component. Can you fix that definition please? Then Marty can finish styling the task details view :+1:

### CODE REVIEW

File: src/components/Pursuance/views/DiscussView.js [hunk 1]
```diff
@@ -1,39 +1,57 @@
...
+              <span className="task-title">{task.title}</span>
             </div>
           </div>
-        </div>
-        <div className="pursuance-task-title-ctn">
-          <div className="pursuance-title-ctn">
-            <span className="pursuance-title">{}</span>
-            <span className="task-title">{}</span>
+          <div className="task-deliverables-ctn">
+            <span><RenderMarkdown content={task.deliverables} /></span>
```
Comment (elimisteve) [inline src/components/Pursuance/views/DiscussView.js:49]: @Moe-Shoman This is the component I'm trying to create

File: src/utils/markdown.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ReactMarkdown } from 'react-markdown';
+
+export const RenderMarkdown = ({ content }) => {
```
Comment (elimisteve) [inline src/utils/markdown.js:4]: @Moe-Shoman This is the component definition that's broken

Comment (Moe-Shoman) [reply src/utils/markdown.js:4]: You did everything right 👍

Comment (Moe-Shoman) [reply src/utils/markdown.js:4]: @elimisteve just didn't import `ReactMarkdown` correctly

Comment (Moe-Shoman) [reply src/utils/markdown.js:4]: remove the curly braces and everything should work fine

**Pull request merged**